### PR TITLE
PSR 368

### PR DESCRIPTION
**Pull request by EricSiegNW**
Add Poptip files
Add basic implementation in Measure VC
Add updated Bridge iOS SDK ref

### CODE REVIEW

File: Psorcast/Psorcast/PopTip/PopTip+Draw.swift [hunk 1]
```diff
@@ -0,0 +1,201 @@
+//
+//  PopTip.swift
+//  AMPopTip
+//
+//  Created by Andrea Mazzini on 01/05/2017.
+//  Copyright © 2017 Andrea Mazzini. All rights reserved.
```
Comment (dephillipsmichael) [inline Psorcast/Psorcast/PopTip/PopTip+Draw.swift:6]: What license is this library under?

File: Psorcast/Psorcast/MeasureTabViewController.swift [hunk 2]
```diff
@@ -88,6 +88,16 @@ open class MeasureTabViewController: UIViewController, UICollectionViewDataSourc
     
     let showInsightTaskId = "showInsight"
     
+    // Poptip related inits
+    // First, should we actually be showing this demo popTip?
+    let showPopTip = true
+    // Now do the inits
+    let popTip = PopTip()
+    var direction = PopTipDirection.up
```
Comment (dephillipsmichael) [inline Psorcast/Psorcast/MeasureTabViewController.swift:96]: Are any of these member variables used besides showPopTip and popTip?

**Comment by dephillipsmichael** [comment]
Looks good to me, but will wait for @DanWebster to review

**Review by DanWebster — APPROVED** [review]
@EricSiegNW Looks good to me.  Still a lot of work to do to establish the where/when for all of these tips, but this does a good job in my opinion of establishing the 'how' that we can build on later

**Pull request merged**